src/utils/user/helpers.js

```diff
@@ -15,6 +17,7 @@ function normalizePublicProfile(user) {
     last_name: _.get(user, 'user_metadata.last_name') || user.family_name,
     phone: _.get(user, 'user_metadata.phone'),
     picture: user.picture,
+    listing_id: _.get(user, 'user_metadata.listing_id'),
```

Comment:
WDYT about having a different method for reporting?
It will help us differentiate between internal dorbel data and public profiles.
listing_id is totally private IMO.